src/components/victory-line.jsx

```diff
@@ -89,6 +90,11 @@ export default class VictoryLine extends React.Component {
      */
     label: React.PropTypes.string,
     /**
+     * The labelComponent prop takes in an entire, HTML-complete label component
+     * which will be used to create labels for line to use
+     */
+    labelComponent: React.PropTypes.element,
+    /**
```

Comment:
I agree with you about not needing label and labelComponent for VictoryLine. I kept it in for consistency since VictoryBar needs both, but maybe that one is the anomaly. I'll look into changing the type for labelComponent to match ReactTransitionGroup.  It makes sense for classes, but wouldn't strings like "div" be confusing to separate from the intended label strings?